contracts/interfaces/IERC20.sol

```diff
@@ -1,14 +0,0 @@
-pragma solidity ^0.5.15;
-
-// ERC20 token interface
-contract IERC20 {
```

Comment:
@vaibhavchellani Why we use the customized IERC20 interface but not the openzepplin's?

Reply:
No reason should switch to openzepplin.